sdk/storage/azure-storage-blob-changefeed/CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 ## Release History
 
+### 12.0.0b5 (Unreleased)
```

Comment:
`12.0.0b4` was already the correct version since it hasn't been released yet. https://pypi.org/project/azure-storage-blob-changefeed/

So, you can just add this feature to the below version and revert the version change for changefeed. We'll just release `12.0.0b4` with STG83